glide.yaml

```diff
@@ -65,32 +68,25 @@ import:
   repo:    https://github.com/openshift/containers-image.git
   version: openshift-3.8
 # pod - sjenning
-# origin-3.11-runc-871ba2e
 - package: github.com/opencontainers/runc
-  repo:    https://github.com/openshift/opencontainers-runc.git
```

Comment:
@soltysh all these removals of the repo lines are intentional?

Reply:
Yes, it's expected and intentional. I've made my due diligence before doing so.

Reply:
I take that back, this change is not necessary, removing it.